.github/workflows/on-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Post-merge chores
```

Comment:
Just use the name "On merge" - I've used names that didn't exactly match the filename and regretted it when I was trying to read things in the github log.